.github/workflows/codeium-pr-agent.yml

```
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+  issue_comment:
+
+jobs:
+  pr_agent_job:
+    if: ${{ github.event.sender.type != 'Bot' }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run pr agent on every pull request, respond to user comments
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: Codium-ai/pr-agent@main
+        env:
+          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+ 
```

```suggestion
        github_action_config.auto_review: "true" # enable\disable auto review
        github_action_config.auto_describe: "true" # enable\disable auto describe
        github_action_config.auto_improve: "true" # enable\disable auto improve
        github_action_config.pr_actions: '["opened", "reopened", "ready_for_review", "review_requested"]'
```

@zonble 如果要自動跑的話要有這些。現在沒有，所以要手動 comment `@CodiumAI-Agent /command`

---

是說我不確定最後一列 `github_action_config.pr_actions: '["opened", "reopened", "ready_for_review", "review_requested"]'` 跟最上層不一致的話會發生什麼事。現在最上層的 `on` 沒有 `"review_requested"`.

---

另外，GitHub App 才支援不加 `@CodiumAI-Agent` 的 `/command`.

---

@zonble Do you want to commit the suggested edits? Since we are still trying out Bito I'm ok with invoking Codium manually, but up to you.